system/services/formbuilder/FormBuilderService.cfc

```diff
@@ -1639,10 +1640,18 @@ component {
 					var mappedColumns = itemColumnMap[ item.id ];
 
 					for( var i=1; i<=mappedColumns.len(); i++ ) {
+						++column;
```

Comment:
While no logical difference I'd go for `column++` here and it would then be consistent with how row is incremented above.

Reply:
Updated increment consistency.